packages/cactus-verifier-client/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./dist",
```

Comment:
@outSH Any chance that you could match the compiler options here used by the rest of the packages?

Reply:
> @petermetz Done, I've also added a single README just now.

@outSH Great, thank you very much for the adjustment!